releasenotes/notes/service-discovery-d7381b098aeb19f6.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Enable language detection via tracers metadata
```

Comment:
Does this really warrant an entry in the release notes? It's not mandatory and usually only used for major changes afaik. If you remove it you can add the `changelog/no-changelog` label to suppress CI complaints.